docs(AUTHORS.rst): Polish contents. (#242)
C4ptainCrunch/ics.py: AUTHORS.rst

* `@Timic3 <https://github.com/Timic3>`_
* `@tomschr <https://github.com/tomschr>`_
* `@Trii <https://github.com/Trii>`_

> * `@tomschr <https://github.com/tomschr>`_

Author: Nice catch!

There are also a few modules or functions incorporated from other
authors and projects:

> * The function ``ics.utils.iso_precision`` includes something like 10 lines of

Reviewer: We can probably drop this paragraph entirely, as the `iso_precision` function is gone together with arrow (we now get the precision directly from reading the `VALUE` type parameter).

Author: Yes!

Reviewer 2: ```suggestion
```

  Arrow's, code, which is written by Chris Smith and under Apache license.

A big part of the code was written at `UrLab <https://urlab.be>`_, an awesome
hackerspace at `ULB <https://ulb.be>`_ in Brussels, Belgium.

Thanks to:

> * `arrow <https://arrow.readthedocs.io/en/latest>`_ which provides a nice API for dates, times and deltas,

Reviewer: Maybe also drop this line, as the API turned out to be not-so-nice for floating datetimes (i.e. without timezone, which is explicitly allowed by the spec). For this reason, `arrow` was completely removed and we're now using plain but correct `datetimes`. See #222.

Author: Yes!

Reviewer 2: ```suggestion
```

> * `python-dateutil <https://dateutil.readthedocs.io/en/stable>`_ for parsing timezones from VTIMEZONE blocks,

Reviewer: ```suggestion
* `python-dateutil <https://dateutil.readthedocs.io/en/stable>`_ for parsing VTIMEZONEs and RRULEs,
```
RRULE support should be coming with #240 (see [here](https://github.com/C4ptainCrunch/ics.py/pull/240/files#diff-aaf03cd716cb3f30c8f8172b896c9719R49)).
Maybe we should also include Tatsu in this list? (#194)

Author: Yes!
